lib/src/formatter.dart

```diff
@@ -254,11 +257,16 @@ class SimpleFormatter implements ReportFormatter {
       ..writeln('${'Timings'.padRight(longestName)}${'ms'.padLeft(pad)}')
       ..writeln(line);
     int totalTime = 0;
-    for (String name in names) {
-      Stopwatch stopwatch = timers[name];
-      totalTime += stopwatch.elapsedMilliseconds;
+    List<_Stat> timings = timers.keys.map((t) {
+      return new _Stat(t, timers[t].elapsedMilliseconds);
+    }).toList();
+    timings.sort();
+    for (_Stat stat in timings) {
```

Comment:
As this is changing behavior, we should add something to the `CHANGELOG`.  (I'll take care of that 👍 )